FastSurferCNN/data_loader/loader.py

```diff
@@ -106,8 +106,7 @@ def get_dataloader(cfg, mode):
             all_augs = {"Elastic": elastic, "Scaling": scaling, "Rotation": rot, "Translation": tl,
                         "RAnisotropy": ra, "BiasField": bias_field, "RGamma": random_gamma}
 
-            all_tfs = {all_augs[aug]: 0.8 for aug in cfg.DATA.AUG if aug != "Gaussian"}
-            gaussian_noise = True if "Gaussian" in cfg.DATA.AUG else False
```

Comment:
This should not be removed as it is important for the data loader during training! This is different from gn_noise during inferende!